package.json

```
@@ -40,7 +40,8 @@
 		"test:update": "wp-scripts test-unit-js --updateSnapshot --config tests/js/jest.config.json",
 		"test:watch": "npm run test -- --watch",
 		"size-check": "bundlewatch",
-		"changelog": "node ./bin/changelog"
+		"changelog": "node ./bin/changelog",
+		"changelog:zenhub": "node ./bin/changelog --changelogSrcType:'MILESTONE'"
```

this should be `ZENHUB_RELEASE`
```suggestion
		"changelog:zenhub": "node ./bin/changelog --changelogSrcType:'ZENHUB_RELEASE'"
```

---

See 77cbc23